front/src/components/signUp/SignUp.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import axios from "axios";
+import ".//signup-style.css";
+import Cookies from "universal-cookie";
+import {
+  validateEmail,
+  validateForm,
+  validatePassword
+} from "../validateCheck/validateForm";
+import AuthApi from "../helpers/authApi";
+import { redirectToUrl, urlUserApi } from "../helpers/baseAPI";
+import {AuthRequest, setCookiesName, setSession, SignUpRequest} from "../helpers/userService";
+import {useFormik} from "formik";
```

Comment:
please use one style everywhere
sometimes you use spaces into {} and sometimes no